css/index.css

```diff
@@ -173,15 +174,114 @@ footer {
     text-align: center;
     margin: 40px 0 20px;
 }
-/*Beginning of Justin's media code*/
+
+/* RWD Solution */
 @media (max-width: 768px) {
```

Comment:
I hope the solution code/lecture was beneficial for your understanding in doing this project. There is definitely a lot of material being chucked at y'all. Also, please don't ever stop asking questions! Highkey - always makes me happy to see you responding and being active in the channels! 😄